### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by lukevers** [lukevers/kittens#33]
As of right now the bot creates directories like the following:

```
./logs/
./logs/{server}/
./logs/{server}/{channel}/
```

But that is all that it currently does.

**Comment by Term1nal** [comment]
As for where.. I'd say detect the GOOS and put it in a canonical location. If *nix, /var/log/kittens. If Windows... working directory where it was run from? "%userprofile%/Docuemnts/kittens/"? Maybe make it a configuration flag to let the user set their log location or whether or not to have logs at all?

**Comment by lukevers** [comment]
Yeah totally; when I made this issue `/var/log` must have slipped my mind. No clue about where windows systems keep their logs, but on *nix systems that would work perfectly.

**Comment by ghost** [comment]
I think that putting it as a command line flag would be the a good option. Then just check to make sure the directory exists and is writable. Let the user specify if/where to store the logs.
